features/project/project_controller.rb

```diff
@@ -160,14 +172,16 @@ class ProjectController < AuthenticatedControllerBase
 
       relative_fastfile_path = nil
       available_lanes = []
-      absolute_fastfile_path = project.local_fastfile_path
-      unless absolute_fastfile_path.nil?
-        parser = Fastlane::FastfileParser.new(path: absolute_fastfile_path)
-        available_lanes = parser.available_lanes
+      # TODO: Follow up in another PR
```

Comment:
Let's create an issue for this, and link is, so we it's easy for people to grab

Reply:
yup, was going to do that after merge 👍

Reply:
https://github.com/fastlane/ci/issues/357